test/const/test_consts.go

```diff
@@ -27,8 +27,8 @@ import (
 
 const (
 	AzureDriverNameVar = "AZURE_STORAGE_DRIVER"
-	TopologyKey        = "topology.disk.csi.azure.com/zone"
-	HostNameLabel      = "kubernetes.io/hostname"
+	TopologyKey        = "topology." + consts.DefaultDriverName + "/zone"
```

Comment:
The static checker is complaining that `consts` is undefined and I see that it is not imported. Is this a missing import, or is this constant defined in this package?